SPECS/libmicrohttpd/libmicrohttpd.spec

```diff
@@ -1,21 +1,29 @@
-Summary:        Lightweight library for embedding a webserver in applications
 Name:           libmicrohttpd
-Version:        0.9.76
-Release:        1%{?dist}
-License:        LGPLv2+
 Vendor:         Microsoft Corporation
-Distribution:   Mariner
-URL:            https://www.gnu.org/software/libmicrohttpd/
+Distribution:   Microsoft Azure Linux
+Version:        0.9.77
+Release:        3%{?dist}
+Summary:        Lightweight library for embedding a webserver in applications
```

Comment:
Please keep the original order of the tags.

Reply:
this is the original order of tags in the spec file we inherit from.